linera-client/src/client_options.rs

```diff
@@ -992,17 +992,17 @@ pub enum DatabaseToolCommand {
         storage_config: String,
     },
 
-    /// List the blobs of the database
+    /// List the blob IDs of the database
```

Comment:
(Maybe "in the database" would be clearer? Also below for chain IDs.)

Reply:
Ok.